docs/source/userguide/command_line.rst

```diff
@@ -460,9 +553,85 @@ Once you've created your script in the appropriate location, follow the command
     geoips test script -p <package_name> <script_name>
     geoips test script --integration <script_name> (no '-p' as this is only supported for 'geoips' package)
 
+.. _geoips_tree:
+
+Tree Command
+------------
+
+:ref:`geoips tree <geoips_tree>`
+
+The GeoIPS CLI provides a variety of commands which aren't necessarily easily exposed
+via ``geoips -h``. To improve this issue, we've added a ``geoips tree`` command which
+exposes all GeoIPS CLI commands in a tree-like fashion. This way, we can expose all
+commands that are available via the GeoIPS CLI, and expose the depth in which these
+commands exist.
+
+By displaying the commands in a depthwise structure, users can understand what commands
+are available and how they are called.
+
+If you just call ``geoips tree``, you'll get the full command tree in a non-colored,
+verbose output.
+
+The output of running ``geoips tree`` is shown below.
+
+.. code-block:: bash
+
+    geoips tree
+
+    geoips
+        geoips config
+            geoips config install
+        geoips get
+            geoips get family
+            geoips get interface
+            geoips get package
+            geoips get plugin
+        geoips list
+            geoips list interface
+            geoips list interfaces
+            geoips list packages
+            geoips list plugins
+            geoips list scripts
+            geoips list test-datasets
+            geoips list unit-tests
+        geoips run
+            geoips run single_source
+            geoips run data_fusion
+            geoips run config_based
+        geoips test
+            geoips test linting
+            geoips test script
+        geoips tree
+        geoips validate
+
+``geoips tree`` additionaly provides optional arguments to filter the output of this
+command. Shown below are these optional arguments and descriptions of what each argument
+does.
+
+* ``--colored``
```

Comment:
This was changed to --color at some point right?

Reply:
Yes. See comment above. Happy to change now or in a final PR that updates the documentation.